presto-docs/src/main/sphinx/functions/binary.rst

```diff
@@ -61,3 +61,7 @@ Binary Functions
 .. function:: xxhash64(binary) -> varbinary
 
     Computes the xxhash64 hash of ``binary``.
+
+.. function:: crc32(binary) -> varbinary
```

Comment:
Why the result is varbinary rather than numeric data type?
At least the commit message hints i should expect number here:

> the result of the function should be read as a long.